src/components/Toolbar/Toolbar.jsx

```diff
@@ -30,6 +30,11 @@ function Toolbar() {
       })
     );
   }
+  function closeNav(){
```

Comment:
I know i'm one who is usually against flat arrow functions but this would look so much cleaner if it was written as such. @Fabricevladimir you think we can have it for a situation like this?

```
  const closeNav = () => {return isOpen ? handleToggle() : null};
```

or if anything we could simplify the logic to:

```
  function closeNav() {
    return isOpen ? handleToggle() : null;
  }
```

Reply:
also can you add white space above for the sake of formatting.